src/main/java/no/nav/veilarbvedtaksstotte/config/CacheConfig.java

```diff
@@ -56,14 +55,6 @@ public Cache veilederNavnCache() {
                 .build());
     }
 
-    @Bean
```

Comment:
Uten å vite helt hvordan det funker – burde vi ha noe caching av arbeidssøkerregisteret?

Reply:
Denne var ikke i bruk – derfor er den fjernet her. Hva gjelder å eventuelt legge inn for Arbeidssøkerregisteret så ser jeg ikke at det det ligger noe for cv, behovsvurdering. En grunn kan jo være at om vi har en cache så er det plutselig ikke er "øyeblikksbilde"? hehe . Usikker her. Kan eventuelt lage en trellooppgave på å legge inn for alle kildene til øyeblikksbilde?

Reply:
Ah, skjønner! Ja, enig i at det kanskje ikke burde caches egentlig 🤔 Hvis vi ser at vi henter ting for ofte, når vi lanserer, så kan vi jo vurdere caching da? Eller i hvert fall se på hvor ofte vi henter data